lib/rubocop/cop/style/if_unless_modifier.rb

```diff
@@ -64,6 +65,13 @@ def autocorrect(node)
 
           ->(corrector) { corrector.replace(node.source_range, oneline) }
         end
+
+        protected
```

Comment:
I'm used to make methods `protected` when I don't want them on the class API but might be useful for subclasses, not really attached to that (I think is what remains of a past as a java developer), let me know what you would prefer.